modules/stanford_news/stanford_news.module

```diff
@@ -67,6 +67,89 @@ function stanford_news_page_attachments(array &$attachments) {
   $attachments['#attached']['library'][] = 'stanford_news/news_node';
 }
 
+/**
+ * Implements hook_views_pre_view().
+ *
+ * Hide the Related Spotlights view if there are fewer than 4 spotlight nodes.
+ */
+function stanford_news_views_pre_view(ViewExecutable $view, $display_id, array &$args) {
```

Comment:
Even if this were the correct solution, the view PR is in soe_profile. The function would not be useful for stanford_profile.

But even so, this is not the way we do things. We don't hard code such functionality into views. The needs to be either re-built or the requirements need to be changed.
1. the filtering needs to be in the view, not some random function like this
2. why hide if there's less than 4 anyways? we don't do this anywhere
3. Even if we do this normally, A quick 30 second google search provides more simple solutions that are about 2 lines long. But such a thing should really be done in the views configuration using either a module or some other approach.